src/acquisition/covidcast/database.py

```diff
@@ -275,6 +275,7 @@ def compute_covidcast_meta(self, table_name='covidcast', use_index=True):
         ROUND(STD(`value`),7) AS `stdev_value`,
         MAX(`value_updated_timestamp`) AS `last_update`,
         MAX(`issue`) as `max_issue`,
+        MIN(`issue`) as `min_issue`,
```

Comment:
We want min over all issues for this source+signal, not just the ones that are marked latest. This will probably need to be done as a separate query, since all the other stats should only apply to the latest issues.

Reply:
Ok! Me and Tara will go through to find out what can we do to this problem.